sdk/core/core-https/src/policies/exponentialRetryPolicy.ts

```diff
@@ -73,9 +80,9 @@ export function exponentialRetryPolicy(
   function shouldRetry(statusCode: number | undefined, retryData: RetryData): boolean {
     if (
       statusCode === undefined ||
-      (statusCode < 500 && statusCode !== 408) ||
-      statusCode === 501 ||
-      statusCode === 505
+      (statusCode < 500 && retryHttpStatusCodes.indexOf(statusCode) === -1) ||
```

Comment:
Since this is in `core-https`, let's use Array.includes here:

```suggestion
      (statusCode < 500 && retryHttpStatusCodes.includes(statusCode) === false) ||
```

Reply:
We chatted over Teams and we agreed on: `&& !retryHttpStatusCodes.includes(statusCode))`.